Changes.md

```diff
@@ -1,3 +1,11 @@
+0.61.x.x (relative to 0.61.4.0)
+========
+
+Breaking Changes
```

Comment:
We can't really bill this as a breaking change if we're including it in `0.61`. Maybe we put something in the Fixes about MessageWidget no longer triggering arbitrary code execution, and then move the comment below to the API section?

Reply:
I've made this change.  I left it a bit late to test it though ... I was expecting it to be easy, but I've been unable to get any messages showing in `gaffer dispatch -gui -tasks GafferDispatch.PythonCommand -show PythonCommand` in either Gaffer 0.61 or 0.62.  I was initially testing in 0.62, and got as far as seeing that `excValue.message` doesn't seem to be working any more in DispatchDialogue, and I had to use `str()` instead, but after sorting that out and fixing my exception, I still haven't been able to see any messages printed.  Maybe I'm doing something wrong?

Reply:
Just checking, did you see the "and operates in immediate mode" part of my recipe for testing?

![image](https://user-images.githubusercontent.com/1133871/159248367-20df44ce-c185-4abb-ae62-7e501ed2dddb.png)

Reply:
You were totally right about me having missed the "immediate mode" part of your recipe.  After getting that right and testing, I get messages, but it's still not running on the UI thread - the ongoing messages update correctly during the run with or without this change.

Reply:
Ah, right, I guess DispatchDialogue does the work in a background thread anyway. So you'd need `gaffer op -gui 1` with an appropriate op, or some custom code to test. I've merged anyway, since I'm convinced the change is correct...